packages/components/src/components/popup/popup.stories.ts

```diff
@@ -5,16 +5,19 @@ import { withActions } from '@storybook/addon-actions/decorator';
 const { argTypes, args, parameters } = storybookDefaults('sd-popup');
 const { generateTemplate } = storybookTemplate('sd-popup');
 
+/**
+ * Popup is a utility that lets you declaratively anchor "popup" containers to another element.
+ */
+
 export default {
```

Comment:
I get an error in the console everytime I open this story, because there is no anchor element.
May we can provide sth. per default?

![image](https://github.com/solid-design-system/solid/assets/3873797/39ed9107-bcd5-4b55-a1f5-8a027feaac3f)

Reply:
Sounds good! I'll do.